src/Components/Facility/AddBedForm.tsx

```diff
@@ -205,7 +224,29 @@ export const AddBedForm = (props: BedFormProps) => {
                     errors={errors.bedType}
                   />
                 </div>
-
+                {!bedId && (
+                  <div>
+                    <div>
+                      <LegacyCheckboxField
+                        label="Do you want to make multiple beds?"
+                        checked={multipleBeds}
+                        onClick={() => setMultipleBeds(!multipleBeds)}
+                      />
```

Comment:
Use `CheckBoxFormField` instead. Avoid using legacy components in new pieces of code.

Reply:
Okay sure, I'll change this.